client/index.ejs

```diff
@@ -1,66 +1,44 @@
 <!DOCTYPE html>
 <html ng-app="app" class="layout-pf layout-pf-fixed layout-pf-fixed-inner-scroll transitions">
 <head lang="en">
-  <title ng-bind="title" translate>Self Service</title>
-  <meta charset="utf-8"/>
-  <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1"/>
-  <meta name="viewport"
-        content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"/>
-  <link rel="shortcut icon" href="/favicon.ico"/>
-  <link rel="icon" href="/favicon.ico"/>
-  <base href="/self_service/">
-  <style>
-    [ng-cloak] {
-      display: none;
-    }
-
-    [ng-cloak].splash {
-      display: block !important;
-    }
-
-    .splash {
-      display: none;
-      background-color: #428bca;
-    }
-  </style>
-
-  <!-- build:css styles/lib.css -->
-  <!-- bower:css -->
-  <!-- endbower -->
-  <!-- endbuild -->
-
-  <!-- build:css styles/app.css-->
-  <!-- inject:css -->
-  <!-- endinject -->
-  <!-- endbuild -->
-
-  <!-- <%- include styles.html %> -->
+    <title ng-bind="title" translate>Self Service</title>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1"/>
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no"/>
+    <link rel="shortcut icon" href="/favicon.ico"/>
+    <link rel="icon" href="/favicon.ico"/>
+    <base href="/self_service/">
+    <style>
```

Comment:
Is this style section needed?  I thought (I could be wrong) that it was in the CSS files?